sizefashion/tsconfig.json

```diff
@@ -0,0 +1,58 @@
+{
+  "compilerOptions": {
+    "target": "es5",
+    "lib": [
+      "dom",
+      "dom.iterable",
+      "esnext"
+    ],
+    "allowJs": true,
+    "skipLibCheck": true,
+    "strict": true,
+    "forceConsistentCasingInFileNames": true,
+    "noEmit": true,
+    "esModuleInterop": true,
+    "module": "esnext",
+    "moduleResolution": "bundler",
+    "resolveJsonModule": true,
+    "isolatedModules": true,
+    "jsx": "preserve",
+    "incremental": true,
+    "plugins": [
+      {
+        "name": "next"
+      }
+    ],
+    "paths": {
+      "@components/*": [
+        "./app/components/*"
+      ],
+      "@context/*": [
+        "./app/context/*"
+      ],
+      "@interfaces/*": [
+        "./app/interfaces/*"
+      ],
+      "@service/*": [
+        "./app/service/*"
+      ],
+      "@assets/*": [
+        "./public/assets/*"
+      ],
+      "@/*": [
+        "./*"
```

Comment:
@mbdesousa you declared the paths and everything but... you're not using them? :(

Reply:
Haven't I?